recipes/openssl/3.x.x/conanfile.py

```diff
@@ -714,10 +714,13 @@ def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "OpenSSL")
         self.cpp_info.set_property("cmake_target_name", "OpenSSL")
         self.cpp_info.set_property("pkg_config_name", "openssl")
-
+        self.cpp_info.names["cmake_find_package"] = "OpenSSL"
+        self.cpp_info.names["cmake_find_package_multi"] = "OpenSSL"
         self.cpp_info.components["ssl"].builddirs.append(self._module_subfolder)
+        self.cpp_info.components["ssl"].build_modules["cmake_find_package"] = [self._module_file_rel_path]
         self.cpp_info.components["ssl"].set_property("cmake_build_modules", [self._module_file_rel_path])
```

Comment:
seems to be required at global scope since 1.43.2: https://github.com/conan-io/conan/issues/10282